tests/generate_fixtures/generate_target_dnadiff_matrices.py

```diff
@@ -104,9 +104,7 @@ def parse_dnadiff_report(dnadiff_report: Path) -> tuple[int, Decimal, Decimal]:
 sim_errors = pd.DataFrame(index=sorted_hashes, columns=sorted_hashes)
 
 # Appending information to matrices
-report_files = Path("../fixtures/fixtures/viral_example/intermediates/dnadiff/").glob(
-    "*.report"
-)
+report_files = Path("../fixtures/viral_example/intermediates/dnadiff/").glob("*.report")
```

Comment:
Good catch! My fault.

Reply:
No worries. When I regenerated fixtures with `make fixtures`, it broke some tests and I was wondering was happened, but managed to find out that it was just a small typo there.